packages/api/src/controllers/timeNotificationController.js

```diff
@@ -0,0 +1,102 @@
+/*
+ *  Copyright 2019-2022 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+const UserFarmModel = require('../models/userFarmModel');
+const TaskModel = require('../models/taskModel');
+const NotificationUser = require('../models/notificationUserModel');
+
+const timeNotificationController = {
+  /**
+   * Notifies farm management of unassigned tasks due this week
```

Comment:
According to @smattingly the first sentence of every jsdoc comment needs to end with a period